Snakefile

```diff
@@ -693,3 +707,23 @@ rule plot_network:
         "logs/plot_network/elec_s{simpl}_{clusters}_ec_l{ll}_{opts}_{attr}_{ext}.log",
     script:
         "scripts/plot_network.py"
+
+
+rule build_test_configs:
+    input:
+        base_config="config.tutorial.yaml",
+        update_file_list=[
+            "test/config.standard.yaml",
```

Comment:
Nice code! I think now it is almost done :) I'd have a last question.

Why do we need the test/config.standard.yaml?
I see in the code that the tutorial being merged with the config.standard.yaml is the standard for all files and there is the special case update_file_list[0] that is treated differently than the rest.
Could we make the config.tutorial.yaml the base configuration and get rid of the test/config..standard.yaml?
That may remove the special case of the [0] and make the code cleaner in my opinion

Reply:
Well spotted. Yes the standard was not necessary in case we test based on the tutorial immediately :)